client/db/bolt/db.go

```diff
@@ -2075,6 +2076,40 @@ func (db *BoltDB) DeleteInactiveMatches(ctx context.Context, olderThan *time.Tim
 	return nil
 }
 
+// SaveDisabledRateFetchers updates disabled exchange rate fetchers.
+func (db *BoltDB) SaveDisabledRateFetchers(disabledFetchers []string) error {
+	return db.Update(func(tx *bbolt.Tx) error {
+		bkt := tx.Bucket(appBucket)
+		if bkt == nil {
+			return fmt.Errorf("failed to open %s bucket", string(appBucket))
+		}
+		disabledByte := []byte(strings.Join(disabledFetchers, ","))
+		return bkt.Put(disabledRateFetchersKey, disabledByte)
+	})
+}
+
+// DisabledRatefetchers retreives a map of disabled exchange rate fetchers.
+func (db *BoltDB) DisabledRatefetchers() (disabledFetchers []string, err error) {
```

Comment:
In other places Fetcher will be capitalized, so I suggest `DisabledRateFetchers` here as well.